libtransmission/verify.cc

```diff
@@ -75,7 +75,6 @@ void tr_verify_worker::verify_torrent(Mediator& verify_mediator, bool const abor
             {
                 bytes_this_pass = num_read;
                 sha->add(std::data(buffer), bytes_this_pass);
-                tr_sys_file_advise(fd, file_pos, bytes_this_pass, TR_SYS_FILE_ADVICE_DONT_NEED);
```

Comment:
With this missing the file to be verified will remain mostly (and needlessly) in OS cache while other data may be removed. Hopefully it will not be released in 4.1.0 as is.